### PR TITLE
use separate display_data to display status div

### LINKED ISSUE #6809: Inline, Minified Resources do not work in classic notebooks
This is due to an interaction with the classic notebooks use of JQuery, when output is published as `text/html`. New notebook code published a div and a script together as `text/html`. Propose to solve by publishing a single script as `application/javascript` (which should work) that creates the necessary div itself

### PR DESCRIPTION
**Pull request by bryevdv**
- [x] issues: fixes #6809
- [ ] tests added / passed

### CODE REVIEW

File: bokeh/core/_templates/autoload_nb_js.js [hunk 1]
```diff
@@ -138,7 +140,9 @@
   function display_loaded() {
     if (root.Bokeh !== undefined) {
       var el = document.getElementById({{ elementid|json }});
-      el.textContent = "BokehJS " + Bokeh.version + " successfully loaded.";
+      if (el !== undefined) {
```
Comment (mattpap) [inline bokeh/core/_templates/autoload_nb_js.js]: `getElementById()` returns `null` if element is not found. You should use `el != null` idiom here.

**Comment by bryevdv** [comment]
@mattpap Thanks, FYI see flowdock for plans re: this miss and getting a release out

**Comment by canavandl** [comment]
This PR causes ``undefined`` to be printed in the output_notebook cell in Jupyter Lab. This could presumably be handled in the JLab extension though.

screen shot 2017-08-25 at 10 45 10 am

**Comment by canavandl** [comment]
And apparently a file save error in JLab (presumably related to the ``undefined`` div)

screen shot 2017-08-25 at 10 46 50 am

**Comment by bryevdv** [comment]
@canavandl any ideas about the errors above?

**Pull request merged**